packages/neos-ui-redux-store/src/UI/ContentCanvas/index.js

```diff
@@ -13,6 +13,7 @@ const SET_PREVIEW_URL = '@neos/neos-ui/UI/ContentCanvas/SET_PREVIEW_URL';
 const SET_SRC = '@neos/neos-ui/UI/ContentCanvas/SET_SRC';
 const FORMATTING_UNDER_CURSOR = '@neos/neos-ui/UI/ContentCanvas/FORMATTING_UNDER_CURSOR';
 const SET_CURRENTLY_EDITED_PROPERTY_NAME = '@neos/neos-ui/UI/ContentCanvas/SET_CURRENTLY_EDITED_PROPERTY_NAME';
+const DOCUMENT_INITIALIZING = '@neos/neos-ui/UI/ContentCanvas/DOCUMENT_INITIALIZING';
```

Comment:
Hey André!
Really nice to see you so active here!
Technically this PR works just as I would imagine it, great job!
One nitpick I have is with naming. Redux actions originally were supposed to represent something like commands, e.g. `FOCUS_PROPERTY` or `SET_SRC`. Then after we started using Redux Sagas we started to tolerate using Redux actions as a kind of events system, that represent that something has already happened, e.g. `DOCUMENT_INITIALIZED`. Those kinds of actions have nothing in their reducers, i.e. they do not chance the state usually, but are merely used to trigger some sage.
The `DOCUMENT_INITIALIZING` action that you added however doesn't feel right, it is actually a command to set document as loading, and I would rename it to something to reflect it better, e.g. `COMMENCE_DOCUMENT_INITIALIZATION` or `SET_DOCUMENT_AS_LOADING`, if you treat it merely as a setter.
Wdyt?